### DISCUSSION TITLE
Specify a different separator for "jpath" passed to call-back options

### DISCUSSION THREAD

**Discussion by YvesMarcoux** [NaturalIntelligence/fast-xml-parser#463] [Ideas]
Some options (like isArray) are call-backs that take a jpath argument. The separator in such paths seems to be the dot (.).
The problem is that the dot is legitimate in XML names, in particular element names. Thus, a jpath of "a.b.c" could mean either "a.b/c" or "a/b.c" (or, of course "a/b/c"). [Here, I use "/" as the delimiter, which BTW would be a good choice, because it is forbidden in XML names.]

Hence, my question: is it possible to specify a separator different from the dot?

Thank you!

**Comment by amitguptagwl** [comment]
Hi @YvesMarcoux 

This was always on top of my priorities but kept getting delayed. But finally, I'm releasing v5.5.0 to handle it gracefully. I hope this will be helpful.
